yargs.js

```diff
@@ -173,41 +174,49 @@ function Yargs (processArgs, cwd, parentRequire) {
   }
 
   self.boolean = function (keys) {
+    argsert('<array|string>', [].slice.call(arguments))
```

Comment:
definitely a bit ugly, but in our discussion in `coding` it sounded like there's a pretty big performance hit with passing in `arguments` -- I opted for ugly and fast.